### PR TITLE
Shows allowed files when uploading in fileed

### LINKED ISSUE #6591: [Suggestion] fileed.php - Show what file types you can upload (Course Local File)?
Right now there is no information on what types of files you can upload. For a better user experience it might be good to display what types of files you are allowed to upload. For example, you are not allowed to upload .cer files but you can upload pdf files.

Screenshot from 2019-05-03 13-17-51

### PR DESCRIPTION
**Pull request by a17adasy**
Fixes #6591
A box shows up containg the allowed files. I am not sure if there are more files allowed but that is easy to add later.

image

This is in the fileed part of dugga when uploading a file.
Test site: https://a17adasy.gruppb.webug.his.se:20002/DuggaSys/fileed.php?cid=2&coursevers=97732

### CODE REVIEW

File: Shared/css/style.css [hunk 1]
```diff
@@ -1715,6 +1715,47 @@ input.hide-moments-button {
   float: left;
 }
 
+.filePopUp {
+  position: relative;
+}
+.fileUploadInfo {
+  display: none;
+  width: 100px;
```
Comment (c17alepe) [inline Shared/css/style.css]: test

File: Shared/css/style.css [hunk 2]
```diff
@@ -1715,6 +1715,47 @@ input.hide-moments-button {
...
+  position: absolute;
+  background: white;
+  padding: 10px 10px 0;
```
Comment (c17alepe) [inline Shared/css/style.css]: It looks a good. I have some suggestion code for a little more centered look. The box wasnt centered when the text is english: 
image


```suggestion
  padding: 10px 10px 0;
  border-radius: 5px;
  text-align: center;
  left: 5%;
```

Comment (c17alepe) [reply Shared/css/style.css]: A quick fade in could be nice too, but not necessary

**Comment by a17adasy** [comment]
A fadein would probably be nicer, will add it now!

**Comment by a17adasy** [comment]
changes added and are ready for another test @c17alepe

**Pull request merged**